tests/h/util/markdown_test.py

```diff
@@ -12,78 +12,70 @@ def test_it_ignores_math_block(self):
         actual = markdown.render("$$1 + 1 = 2$$")
         assert actual == "<p>$$1 + 1 = 2$$</p>\n"
 
-    def test_it_ignores_inline_match(self):
+    def test_it_ignores_inline_math(self):
         actual = markdown.render(r"Foobar \(1 + 1 = 2\)")
         assert actual == "<p>Foobar \\(1 + 1 = 2\\)</p>\n"
 
-    def test_it_sanitizes_the_output(self, markdown_render, sanitize):
-        markdown.render("foobar")
-        sanitize.assert_called_once_with(markdown_render.return_value.return_value)
-
-    @pytest.fixture
-    def markdown_render(self, patch):
-        return patch("h.util.markdown._get_markdown")
-
-    @pytest.fixture
-    def sanitize(self, patch):
-        return patch("h.util.markdown.sanitize")
```

Comment:
This test and these two patch fixtures were used to test that `render()` calls `_sanitize()` which is no longer necessary now that all the tests go through `render()`